roles/mailserver/files/docker-compose.yml

```diff
@@ -94,6 +94,34 @@ services:
       - /mnt/docker/php/custom.ini:/usr/local/etc/php/conf.d/custom.ini
     environment:
       - WORDPRESS_DB_PASSWORD={{ db_password }}
+{% else %}
+  # Contact Form (Optional)
+  # https://github.com/ksylvan/Simple-PHP-Contact-Form
+  contact:
+    image: kayvan/contact-form
+    container_name: contact
+    # restart: always
+    volumes:
+      - /mnt/docker/contact/config:/contact/config
+      - /mnt/docker/contact/ssmtp:/etc/ssmtp
+      - /mnt/docker/contact/index.php:/contact/index.php
+    depends_on:
+      - mailserver
+{% endif %}
+
```

Comment:
The whole else-endif block is using the wrong level of indentation. This whole block should not show up as a change at all at this point.